contrib/pkg/adm/managedns/enable.go

```diff
@@ -96,8 +98,9 @@ func NewEnableManageDNSCommand() *cobra.Command {
 	}
 
 	flags := cmd.Flags()
-	flags.StringVar(&opt.Cloud, "cloud", cloudAWS, "Cloud provider: aws(default)|gcp)")
+	flags.StringVar(&opt.Cloud, "cloud", cloudAWS, "Cloud provider: aws(default)|gcp|azure)")
 	flags.StringVar(&opt.CredsFile, "creds-file", "", "Cloud credentials file (defaults vary depending on cloud)")
+	flags.StringVar(&opt.AzureResourceGroup, "azure-resource-group", "os4-common", "Azure Resource Group (Only applicable if --cloud azure)")
```

Comment:
To stay consistent with the flag used in `hiveutil create-cluster`, rename this flag to `azure-resource-group-name`.
```suggestion
	flags.StringVar(&opt.AzureResourceGroup, "azure-resource-group-name", "os4-common", "Azure Resource Group (Only applicable if --cloud azure)")
```